roles/foreman_repositories/tasks/main.yml

```diff
@@ -9,3 +9,9 @@
   tags:
     - packages
   when: ansible_os_family == 'RedHat'
+
+- name: 'Clean yum'
```

Comment:
Could this be configurable with a variable? It's really useful for some cases, but others, like in devel, I might install other packages manually and would benefit from the cache.

Reply:
Can you expand further the benefits of keeping the cache when setting up foreman_repositories?

Reply:
Keeping the yum cache makes installing packages faster than cleaning the cache, no?

Reply:
It does, but in this specific case we need this for things to be correct.

Reply:
What exactly do you mean 'for things to be correct'?

Reply:
If you read my original comment, you'll see that I do not know why it started being broken but this does fix things because we are installing a release that alters the yum repository definition. Yum appears to not be updating the cache and so we need to clean it for things to work properly.

Reply:
Why not do the yum clean in the pipeline then? Or `yum clean all --enablerepo="foreman" --disablerepo="*"`, if EPEL and the main OS repo already have a valid cache, it's really time-consuming to regenerate it.